EDEngineer/Utils/UI/PersistGroupExpandedStateBehavior.cs

```diff
@@ -73,6 +73,9 @@ private void OnExpanded(object sender, RoutedEventArgs e)
 
         private void SetExpanded(bool expanded)
         {
+            if (GroupName == null)
+                return;
```

Comment:
could you add the brackets, even if it's a single line under the if?

Reply:
Done